components/profile_popover/profile_popover.jsx

```diff
@@ -299,7 +301,7 @@ class ProfilePopover extends React.PureComponent {
             expiryContent = customStatusSet && customStatus.expires_at && customStatus.duration !== CustomStatusDuration.DONT_CLEAR && (
                 <span>
                     {' (Until '}
-                    {displayExpiryTime(customStatus.expires_at, this.props.timezone)}
+                    {this.displayExpiryTime(customStatus.expires_at, this.props.timezone)}
                     {')'}
                 </span>
             );
```

Comment:
If this is repeated a couple times, maybe you can refactor this to a common component